CMakeLists.txt

```diff
@@ -38,23 +38,50 @@ if (VUL_IS_TOP_LEVEL)
     endif()
 
     include(GNUInstallDirs)
+    include(CMakePackageConfigHelpers)
 
     install(
         DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/include/${API_TYPE}/"
         DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/vulkan"
     )
 
+    # Create VulkanUtilityLibraries-targets.cmake
     set_target_properties(VulkanLayerSettings PROPERTIES EXPORT_NAME "LayerSettings")
     set_target_properties(VulkanUtilityHeaders PROPERTIES EXPORT_NAME "UtilityHeaders")
     set_target_properties(VulkanCompilerConfiguration PROPERTIES EXPORT_NAME "CompilerConfiguration")
     install(
         TARGETS VulkanLayerSettings VulkanUtilityHeaders VulkanCompilerConfiguration
-        EXPORT VulkanUtilityLibrariesConfig
+        EXPORT VulkanUtilityLibraries-targets
         INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
     )
     install(
-        EXPORT VulkanUtilityLibrariesConfig
+        EXPORT VulkanUtilityLibraries-targets
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/VulkanUtilityLibraries
         NAMESPACE Vulkan::
     )
+
+    # Create VulkanUtilityLibrariesConfig.cmake
+    set(VUL_EXPORT_TARGETS ${CMAKE_INSTALL_LIBDIR}/cmake/VulkanUtilityLibraries/VulkanUtilityLibraries-targets.cmake)
+    file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/VulkanUtilityLibrariesConfig.cmake.in" [=[
+        @PACKAGE_INIT@
+
+        include(CMakeFindDependencyMacro)
+        # NOTE: Because VulkanHeaders is a PUBLIC dependency it needs to be found prior to VulkanUtilityLibraries
+        find_dependency(VulkanHeaders REQUIRED)
```

Comment:
find_package is called the under the hood here for the user.

Reply:
This also gives the flexibility to check either `find_package(Vulkan)` or `find_package(VulkanHeaders CONFIG)`

FWIW